### ISSUE TITLE
empty recording stream with branches 7.2.1 and master

### ISSUE THREAD

**Issue by juha-h** [sipwise/rtpengine#749]
Recording works fine at least with branches 7.0.1 and 7.1.2, but when rtpengine branch is 7.2.1 or master, recording stream file xxx-mix.mp3 contains only audio file header and no data.

In all tests I have used the same configuration files.  Recording configuration file is this:
```
[rtpengine-recording]
table = 0
output-storage = db
output-format = mp3
output-mixed = 1
spool-dir = /var/spool/rtpengine
output-dir = /var/spool/rtpengine/tmp
mysql-host = 127.0.0.1
mysql-port = 3306
mysql-user = xxx
mysql-pass = xxx
mysql-db = rtpengine
```
All tests have been done using Debian Stretch rtpengine packages.

**Comment by rfuchs** [comment]
Can't reproduce this, using PCMU and identical config (except no db storage set). Do you have a pcap?

**Comment by rfuchs** [comment]
Fixed via #721
